go/vt/vttablet/tabletserver/schema/engine.go

```diff
@@ -178,10 +206,17 @@ func (se *Engine) EnsureConnectionAndDB(tabletType topodatapb.TabletType) error
 	ctx := tabletenv.LocalContext()
 	conn, err := dbconnpool.NewDBConnection(ctx, se.env.Config().DB.AllPrivsWithDB())
 	if err == nil {
+		log.Infof("inside EnsureConnectionAndDB with TabletType %v...", tabletType)
```

Comment:
I'm assuming we'll be removing messages like this before merging?

Reply:
yes